content/examples/databricks-cluster.Rmd

```diff
@@ -0,0 +1,83 @@
+---
+title: "Using sparklyr with Databricks"
+output:
+  html_document:
+    toc: true
+    toc_depth: 2
+    toc_float:
+      collapsed: false
+params:
+  width: 600
+---
+
+## Overview
+
+This documentation demonstrates how to use `sparklyr` with RStudio Server Pro
+and a Databricks cluster with Apache Spark.
+
+## Best practices
+
+- **Connect to Spark remotely** - Install RStudio Server Pro outside of the
+  Databricks cluster and work with Spark remotely
+- **Work across RStudio Team** - Install RStudio Connect and RStudio Package
+  Manager outside of the Databricks cluster so that they are not limited to the
+  resources or ephemeral nature of Databricks clusters
+- **Restrict workloads to interactive analysis** - Only perform workloads
+  related to exploratory or interactive analysis with Spark, then write the
+  results to a database, file system, or cloud storage for more efficient
+  retrival in apps, reports, and APIs
+- **Load and query results efficiently** - Shiny apps that use Spark on the
+  backend tend to have performance and runtime issues; instead, consider reading
+  the results from a database, file system, or cloud storage instead
```

Comment:
```suggestion
- **Load and query results efficiently** - Shiny apps that use Spark on the
  backend tend to have performance and runtime issues; instead, consider reading
  the results from a database, file system, or cloud storage, as outlined in the previous point.
```

Reply:
Regardless of whether this suggestion is accepted or not, the second `instead` should be dropped from the original sentence.

Reply:
Fixed in 6c929e3 and added more commentary on Shiny + Spark issues.